src/manager.js

```diff
@@ -270,3 +272,22 @@ Manager.prototype.processOnEnd = function (evt) {
     var index = self.nipples.indexOf(nipple);
     self.nipples.splice(index, 1);
 };
+
+// Remove destroyed nipple from the list
+Manager.prototype.ondestroyed = function(evt, nipple) {
+    nipple = this.nipples.get(nipple.identifier);
```

Comment:
One issue here, and might be the case for all events: the nipple instance passing along is not the same instance in `nipples` manager list. Got to get the manager side instance by `identifier`. This comes from `Nipple#constructor` returning a custom object.